.gitignore

```diff
@@ -12,6 +12,7 @@
 /install-sh
 /ltmain.sh
 /missing
+/build
```

Comment:
@gjanssens 
1. Can you improve https://wiki.gnucash.org/wiki/Documentation_Update_Instructions and related pages like https://wiki.gnucash.org/wiki/Initializing_Documentation_Build_Environment so that this is no longer required?
2. it seems you had also forgotten to remove the entries before while removing autotools. ;-)

Reply:
@gjanssens
>     1. Can you improve https://wiki.gnucash.org/wiki/Documentation_Update_Instructions and related pages like https://wiki.gnucash.org/wiki/Initializing_Documentation_Build_Environment so that this is no longer required?

https://wiki.gnucash.org/wiki/Translation#Translating_the_GnuCash_Guide_and_Help might be related, too.

Reply:
> @gjanssens
> 
>     1. Can you improve https://wiki.gnucash.org/wiki/Documentation_Update_Instructions and related pages like https://wiki.gnucash.org/wiki/Initializing_Documentation_Build_Environment so that this is no longer required?

Done

> 
>     2. it seems you had also forgotten to remove the entries before while removing autotools. ;-)
> 
I think it's better to keep them around for at least one major release. People with git clones that have been using autotools in the past, will still have those files in their working directories. Removing these entries may cause them to be added accidentally in future PRs or patches.

>     3. [Edit:] can you look over the failed checks?

I believe @jralls took care of that. I don't see any failed checks any more.